src/Metadata.php

```diff
@@ -78,8 +78,8 @@ public function get( $check, $object_id, $meta_key, $single ) {
 				}
 			}
 		}
-		if ( empty( $check ) && $single ) {
-			$check = array( '' ); // Ensure an empty string is returned when meta is not found.
+		if ( empty( $check ) ) {
+			$check = get_metadata_default( $this->meta_type, $object_id, $meta_key, $single );
```

Comment:
```suggestion
			return get_metadata_default( $this->meta_type, $object_id, $meta_key, $single );
```

It is nice to be straight-forward.

Reply:
Done. Thanks